lib/fathead/reactjs/parse.py

```diff
@@ -95,8 +95,15 @@ def parse_example(self, api):
             if tag.name == 'div':
                 code = str(tag.find('pre'))
                 if code != 'None':
-                    example_code = code
-        return example_code.strip('\n')
+                    example_code = tag.text.strip()
+        example_code+='\n'
+        return example_code.replace('\n','\\n')
```

Comment:
@dip-kush looks like we also have instances of multiple newlines (e.g. `\n\n`) We should normalize those to a single newline